realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -16,6 +16,8 @@
 
 package io.realm.internal;
 
+import android.text.TextUtils;
```

Comment:
Please don't include android dependencies in the internal API. It will make it harder to migrate to the JVM.